### PR TITLE
added TurkishMovieSentiment dataset

### PR DESCRIPTION
**Pull request by yavuzKomecoglu**
This PR adds the **TurkishMovieSentiment: This dataset contains turkish movie reviews.**

- **Homepage:** [https://www.kaggle.com/mustfkeskin/turkish-movie-sentiment-analysis-dataset/tasks](https://www.kaggle.com/mustfkeskin/turkish-movie-sentiment-analysis-dataset/tasks)
- **Point of Contact:** [Mustafa Keskin](https://www.linkedin.com/in/mustfkeskin/)

### CODE REVIEW

File: datasets/turkish_movie_sentiment/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [https://www.kaggle.com/mustfkeskin/turkish-movie-sentiment-analysis-dataset/tasks](https://www.kaggle.com/mustfkeskin/turkish-movie-sentiment-analysis-dataset/tasks)
+- **Point of Contact:** [Mustafa Keskin](https://www.linkedin.com/in/mustfkeskin/)
+
+### Dataset Summary
+
+This data set is a dataset consisting of Turkish movie reviews and scored between 0-5.
```
Comment (lhoestq) [inline datasets/turkish_movie_sentiment/README.md:54]: can you add more details please ?

File: datasets/turkish_movie_sentiment/turkish_movie_sentiment.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+import csv
+import logging
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+    This dataset contains turkish movie reviews.
```
Comment (lhoestq) [inline datasets/turkish_movie_sentiment/turkish_movie_sentiment.py:28]: same here

File: datasets/turkish_movie_sentiment/turkish_movie_sentiment.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+"""
+
+_CITATION = ""
```
Comment (lhoestq) [inline datasets/turkish_movie_sentiment/turkish_movie_sentiment.py:31]: There is no citation ?

**Review by lhoestq — COMMENTED** [review]
Really cool thank you !

Only the dataset_infos.json file is missing

You can generate it with
```
datasets-cli test ./datasets/turkish_movie_sentiment --save_infos --all_configs --data_dir <path/to/folder>
```

**Review by lhoestq — APPROVED** [review]
I just generated the dataset_infos.json file

Thanks for adding this one !

**Comment by yavuzKomecoglu** [comment]
> I just generated the dataset_infos.json file
> 
> Thanks for adding this one !

Thank you very much for your support.

**Pull request merged**